libraries/AP_Logger/LogStructure.h

```diff
@@ -2681,6 +2681,7 @@ enum LogMessages : uint8_t {
     LOG_OA_BENDYRULER_MSG,
     LOG_OA_DIJKSTRA_MSG,
     LOG_VISUALVEL_MSG,
+    LOG_RICHENPOWER_MSG,
```

Comment:
Is LOG_RICHENPOWER_MSG used?  It seems a bit like we are using the short-form of writing to the logs that don't require this.  Maybe i'm wrong..

Reply:
Nope.  Appendix removed.